pkg/services/ngalert/api/api_admin.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 type AdminSrv struct {
-	scheduler Scheduler
+	scheduler ExternalAlertmanagerProvider
```

Comment:
Should this still be called the scheduler when what it does is provide Alertmanagers?

Reply:
1. It doesn't anymore. Dispatcher does
2. The interface is declared on the user side which probably don't care about Dispatcher and cares only about some of its methods

Reply:
Renamed